packages/rlc-common/src/metadata/packageJson/buildAzureMonorepoPackage.ts

```diff
@@ -86,6 +86,8 @@ export function getAzureMonorepoDependencies(config: AzureMonorepoInfoConfig) {
         : "workspace:*",
       "@azure/identity": !shouldUsePnpmDep ? "^4.6.0" : "catalog:internal",
       "@types/node": !shouldUsePnpmDep ? "^18.0.0" : "catalog:",
+      "@eslint": !shouldUsePnpmDep ? "^9.9.0" : "catalog:",
+      ...(config.specSource === "Swagger" && { autorest: "latest" }),
```

Comment:
why we have this?

Reply:
@MaryGao It may be historic convention that we just install `autorest` globally.  I don't see any issue with adding it to the catalog.  Should we use "latest" mostly?

Reply:
Yes, for autorest it is mainly for legacy SDKs which is generated from swagger path. I think it would become less and less in that path. 

BTW do you think if we should manage all deps into catalog/workspace in SDK repo?

Reply:
@jeremymeng We plan to use ``` autorest: "catalog:" ``` and merge this pr. So we need to add this dependency into catalog in SDK repo. Do you have any concerns?